Flank/flank
# fix: Enhance table builder logic

Fixes #1893 

## Test Plan
> How do we know the code works?

1. Run `./gradlew flankFullRun`
2. Run `flank firebase test android models list`
3. You should see `OS_VERSION_IDS` column aligned to the left
4. You should see `RESOLUTION` column centered by `x` mark
